docs/source/data/data_catalog.md

```diff
@@ -509,18 +509,18 @@ Making a simple addition to your Data Catalog allows you to perform versioning o
 Consider the following versioned dataset defined in the `catalog.yml`:
 
 ```yaml
-cars.csv:
```

Comment:
Noticed that the naming here was inconsistent with other data catalog examples, including others on the same page. Dropped the file extension to fix that.